### PR TITLE
Button to dismiss setting changes. Resolves issue #20 (probably)

### PR DESCRIPTION
**Pull request by fghber**
First implementation of feature request #20 
I am new to that OSS gitting. So, please give me a sign, if I've done something not as supposed. BTW: Hi there.

### CODE REVIEW

File: YAPA/Settings.xaml.cs [hunk 1]
```diff
@@ -75,6 +79,8 @@ public Settings(IMainViewModel host, double currentOpacity, Brush currentTextCol
 
         private async void Settings_Loaded(object sender, RoutedEventArgs e)
         {
+            this.Activate();
```
Comment (floatas) [inline YAPA/Settings.xaml.cs]: Looks like this line will also fix #42.

File: YAPA/Settings.xaml.cs [hunk 2]
```diff
@@ -115,6 +121,41 @@ public Settings(IMainViewModel host, double currentOpacity, Brush currentTextCol
...
+        /// The MsgBox is not shown, if the Settings window was Saved() or Dismissed()
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        void Settings_Closing(object sender, CancelEventArgs e)
+        {
+#if DEBUG
+            System.Windows.MessageBox.Show("Closing called");
+#endif
+            // If data is dirty, notify user and ask for a response
```
Comment (floatas) [inline YAPA/Settings.xaml.cs]: Not sure if pragma Debug is needed, message box can be replaced with Debug.WriteLine(...)

File: YAPA/Settings.xaml.cs (hunk 1 above)
Comment (fghber) [inline YAPA/Settings.xaml.cs]: Did you mean #45 ? Yes, it does.

File: YAPA/Settings.xaml.cs (hunk 2 above)
Comment (fghber) [inline YAPA/Settings.xaml.cs]: Fixed it as suggested by you.

**Comment by fghber** [comment]
Thank you for mentioning the issue with the jumping MainWindow. Fixed it as suggested by you via a recycled method to save the window position. I also do the same before tray state changes, to fix #27 too.

Regarding the new branch. I do not fully understand the rationale behind this.
Do you mean I should do this: http://stackoverflow.com/questions/1628563/move-the-most-recent-commits-to-a-new-branch-with-git ?

**Comment by floatas** [comment]
Every bug fix should be made in separate branch. Multiple small fixes in multiple branches are much better than big changes in one branch. 
It would be enough if you could create new branch (write in title what it fixes) from your master and make pull request from it. You don't need to do what it says in stackoverflow.

Sorry about that, we should create readme with detailed instructions for contributors.

**Comment by fghber** [comment]
Okay, I've created a new branch fixes and pulled a request again.

**Comment by fghber** [comment]
Should I this PR?

**Comment by floatas** [comment]
I think we can close this PR, I will try to review your other pull request this week

**Pull request closed without merging**